packages/ffe-searchable-dropdown-react/src/index.d.ts

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+interface DropDownItem {
+    [key: string]: any;
+}
+
+interface ListElementBodyProps {
+    item: DropDownItem;
+    isHighlighted: boolean;
+}
+
+interface NoMatch {
+    text: string;
+    dropdownList?: DropDownItem[];
+}
+
+export interface SearchableDropdownProps {
+    id: string;
+    labelId: string;
+    className?: string;
+    dropdownList: DropDownItem[];
+    dropdownAttributes: string[];
```

Comment:
`dropdownAttributes` and `searchAttributes` must be existing keys in `DropDownItem`. Can I achive this with typescript?